webrender/src/picture.rs

```diff
@@ -115,6 +116,27 @@ impl PicturePrimitive {
         }
     }
 
+    pub fn resolve_scene_properties(&mut self, properties: &SceneProperties) -> bool {
```

Comment:
The scope of this method seems a bit inconsistent to me: in includes opacity filters but excludes the transforms. Could we, perhaps, resolve everything in one place (here)?

Reply:
I'm not really sure how, since the transforms are stored in the clip-scroll tree, and the filters are stored in the picture tree?